docker-compose.yml

```
@@ -13,29 +13,24 @@ services:
     - "5000:8081"
     volumes:
     - ./wharf-docker-compose/traefik:/etc/traefik # Traefik configuration file
-  web:
-    image: quay.io/iver-wharf/wharf-web
+#  web:
+#    image: w-web:1
+#    ports:
+#    - "8080:8080"
```

This probably shouldn't be here, right?
```suggestion
  web:
    image: quay.io/iver-wharf/wharf-web
```

---

I think maybe your right.  My crazy debugging stuff will be removed.
We have had the web in there commented out though. Keeping it there commented makes sense with the documentation. 
It may make sense to rethink the port on the web to to satisfy both the requirement of not colliding it with the baremetal `npm run start` and also keeping in compliance with the defaults for CORS when using OIDC. 
My sugestion here would be port **4201** thoughts?